ui/src/app/utils/runtime-hooks.tsx

```diff
@@ -292,7 +299,7 @@ export const useCustomRuntime = (
 
   useEffect(() => {
     let mounted = true;
-    const aborter = new AbortController();
+    aborter = new AbortController();
```

Comment:
same comment: why is this necessary?  it is set in L150

Reply:
This causes interference in the test if its not declared again

Reply:
Yeah the only thing i know is that every time we render useCustome is triggered which calls the useEffect where its reinitializes. We can debug this in mob?